### PR TITLE
VAM-9 Load size

### PR DESCRIPTION
**Pull request by sidjtd**
Async ftw!

### CODE REVIEW

File: lib/broker/models/orwch-loadsiz.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        ],
+        invokeRPC: (args, session, callback) => {
+            if (_.isEmpty(session.userId)) {
+                callback(null, '');
+                return;
+            }
+            // Get the font size via ParameterService
+            testService.loadSize(args, session, callback);
+        },
+        formatResult: res => res,
```
Comment (mfuroyama) [inline lib/broker/models/orwch-loadsiz.js]: So, if the `formatResult` function does nothing, you can omit it...in the Pure JS implementations, these are optional

File: lib/services/testService/loadSize.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+const ParameterService = require('../parameterService');
+
+/**
+ * Get the font size via ParameterService
+ * @param  {Object} context - the session data
+ * @param  {Function} callback - the callback function used to process results
+ */
+function loadSize(args, context, callback) {
+    const name = args[0] || false;
+    ParameterService.withContext(context).get('ORWCH BOUNDS', { entity: 'USR', instance: name})
```
Comment (mfuroyama) [inline lib/services/testService/loadSize.js]: Woof...I hate to be pedantic about things like this, but don't forget the space after `name`...

File: lib/services/testService/loadSize.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        .then((res) => {
+            if (_.isString(name) && _.has(res, 'value')) {
+                let val = (_.isArray(res) && res.length > 0) ? res[0].value : res.value;
+                console.log('cuute', val);
```
Comment (mfuroyama) [inline lib/services/testService/loadSize.js]: Also, even though this debug is 'chou kawaii' you might want to remove this from the final committed code set.  ;)

File: lib/broker/models/orwch-loadsiz.js (hunk 1 above)
Comment (sidjtd) [reply lib/broker/models/orwch-loadsiz.js]: Got it!!!!

File: lib/services/testService/loadSize.js (hunk 3 above)
Comment (sidjtd) [reply lib/services/testService/loadSize.js]: Yesssirrr! <3

**Comment by mfuroyama** [comment]
@sidjtd One more thing to remember is that if you include the Jira issue number in the pull request title, it'll link the pull request to the issue and it'll automatically move the issue to 'In Review'

**Comment by mfuroyama** [comment]
@sidjtd Also, in the near future, let's move the Service code out of the 'testService' and into something like a `orderMetaService` instead. The code though, as it is, looks good.

**Comment by sidjtd** [comment]
Ah, roger on the pull request, I'll actually re-do the other one right now!

**Pull request merged**